k8s-operator/apis/v1alpha1/types_tsrecorder.go

```diff
@@ -50,52 +56,157 @@ type TSRecorderSpec struct {
 	// TODO(tomhjp): Support a hostname or hostname prefix field, depending on
 	// the plan for multiple replicas.
 
-	// TSRecorder image. Defaults to tailscale/tsrecorder, with the same tag as
-	// the operator.
-	// +optional
-	Image Image `json:"image,omitempty"`
-
 	// If enabled, TSRecorder will serve the UI with HTTPS on its MagicDNS hostname.
-	// See --ui flag for more details: https://tailscale.com/kb/1246/tailscale-ssh-session-recording#deploy-a-recorder-node.
+	// Set to true to enable the TSRecorder UI. The UI lists and plays recorded sessions.
+	// The UI will be served at <MagicDNS name of the recorder>:443. Defaults to false.
```

Comment:
Nit: I think this now has the same information twice (also line 59)

Reply:
Thanks for catching, fixed.